sumologic/provider.go

```diff
@@ -74,6 +74,7 @@ func Provider() terraform.ResourceProvider {
 			"sumologic_password_policy":                    resourceSumologicPasswordPolicy(),
 			"sumologic_saml_configuration":                 resourceSumologicSamlConfiguration(),
 			"sumologic_kinesis_metrics_source":             resourceSumologicKinesisMetricsSource(),
+			"sumologic_token":                              resourceSumologicToken(),
```

Comment:
just one concern about the resource name, do we have any other kind of tokens in Sumo? should we name this `sumologic_collector_installation_token` or something more specific?

Reply:
This is a good call. 

Actually, this aligns with our api name

![Screen Shot 2021-06-16 at 4 02 07 PM](https://user-images.githubusercontent.com/11988177/122305823-4c0c9a80-cebc-11eb-8f14-174b24206c87.png)
 (https://nite-api.sumologic.net/docs/#tag/tokensLibraryManagement)

I think our original intention is that if there is any other kind of token in Sumo, we still want to have it use the same token, but change `type` to something else. Our current installation token is using the type of `CollectorRegistration`